NEWS

```diff
@@ -1,5 +1,21 @@
 systemd System and Service Manager
 
+CHANGES WITH 238 in spe:
+
+        * The MemoryAccounting= unit property now defaults to on. After
+          discussions with the upstream control group maintainers we learnt
+          that the negative impact of cgroup memory accounting on current
+          kernels is finally relatively minimal, so that it should be safe to
```

Comment:
I think it would be great if there were a link to graphs, charts or whatever showing what is meant by "relatively small". Also, the phrase "current kernel" seems a little bit vague to me. Would it be more useful to mention specific versions/commits?

Reply:
In terms of kernel versions, we've been enabling it by default since v4.6. We've encountered a couple corner cases which showed scalability issues since then. All fixes were pushed upstream and v4.14+ should be fine.

As for performance numbers, in the aggregate running profile of six digit number of machines running with memcg enabled, the top memcg function which shows up is mm/memcontrol.c::try_charge() at 0.2%, a level which is usually entirely unnoticeable. Plus, we haven't had any noticeable performance regression reported due to memcg from internal benchmarks and production uses except for the above mentioned scalability issues which are already fixed.